alibi_detect/od/sklearn/svm.py

```diff
@@ -0,0 +1,143 @@
+import numpy as np
+from typing import Dict, Union, Callable, Optional
+from typing_extensions import Literal
+from sklearn.linear_model import SGDOneClassSVM
+
+from alibi_detect.od.sklearn.base import SklearnOutlierDetector
+from sklearn.kernel_approximation import Nystroem
+
+
+class SVMSklearn(SklearnOutlierDetector):
+    ensemble = False
+
+    def __init__(
+        self,
+        n_components: Optional[int] = None,
+        kernel: Union[Literal['linear', 'poly', 'rbf', 'sigmoid'], Callable] = 'rbf',
+        sigma: Optional[float] = None,
+        kernel_params: Optional[Dict] = None,
+    ):
+        """SKlearn backend for the One class support vector machine (SVM) outlier detector.
+
+        Parameters
+        ----------
+        n_components
+            Number of features to construct. How many data points will be used to construct the mapping.
```

Comment:
This is a very different description from the `torch` backend and the wrapper classes, please review.

Reply:
Probably applies to other arguments too.

Reply:
👍🏻 , since no longer separate `torch` and `sklearn` backends. Will also be covered by #818